string name
string type
string event
string value_string
float64 value

int8 direction

int8 DIRECTION_UI_TO_APPLICATION = 0
int8 DIRECTION_APPLICATION_TO_UI = 100